src/helpers/generateSlug.js

```diff
@@ -0,0 +1,15 @@
+import slugify from 'slug';
+import uniqid from 'uniqid';
+
+const generateSlug = (title) => {
+  if (title.length > 70) title = title.substring(0, 70);
+  // generate slug;
+  const slug = `${slugify(title, {
```

Comment:
you could just return directly instead of having a variable `slug`

Reply:
Thanks, Noted